django_redshift_backend/base.py

```diff
@@ -279,228 +305,482 @@ def add_field(self, model, field):
         if self.connection.features.connection_persists_old_columns:
             self.connection.close()
 
+    # BASED FROM https://github.com/django/django/blob/3.2.12/django/db/backends/base/schema.py#L611-L864
     def _alter_field(self, model, old_field, new_field, old_type, new_type,
                      old_db_params, new_db_params, strict=False):
```

Comment:
1. I remove previous `_alter_field` once
2. copy the newer method from https://github.com/django/django/blob/3.2.12/django/db/backends/base/schema.py#L611-L864
3. modify for Redshift

So, the part of this diff in this PR is not important.